.github/workflows/test.yml

```diff
@@ -177,6 +177,7 @@ jobs:
         uses: actions/checkout@v4
         with:
           repository: extendr/rextendr
+          ref: use_try_from
```

Comment:
this is a stop gap that will be removed following the fixes to rextendr

Reply:
Yes! Also mentioned in the first post!